plugins/grpc/ruby/v1.62.2/buf.plugin.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/grpc/ruby
+plugin_version: v1.62.2
+source_url: https://github.com/grpc/grpc
+description: Generates Ruby client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/ruby:v25.3
```

Comment:
Should we be bumping the deps, `v26.1` ?

Reply:
Nope - see https://github.com/bufbuild/plugins/pull/1188#discussion_r1570942405.